### DISCUSSION TITLE
[Feature] Trust system CA certificates/self signed certs

### DISCUSSION THREAD

**Discussion by hendrikreuhl** [immich-app/immich#2375] [Feature Request]
### The feature

Hey,

first of all, thanks for all the development. I searched a long time for an alternative like this and i love it.

It would be great if the app would trust system ca certificates.
In my example, im running immich in a local enviroment and connect to it via vpn, as i don't wont to expose it to the www.
Since I do not use ACME but a self signed cert instead, it would be great to have the option to trust self signed certs or follow the system ca's like nextcloud etc. do

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
There's no good way to do this in flutter. See #2139 and #765.

You can use ACME via DNS without exposing your services. I recommend that approach.

**Comment by jrasm91** [comment]
Android let's you install a user provided certificate at the operating system level. Does this not work you?

**Reply by bo0tzz** [reply]
Flutter/dart uses its own certificate store

**Comment by hendrikreuhl** [comment]
Thanks a lot, didn't think about that. Thanks for your reply!
